cmd/kubeadm/app/constants/constants.go

```diff
@@ -27,6 +27,10 @@ const (
 	// KubernetesDir is the directory kubernetes owns for storing various configuration files
 	KubernetesDir = "/etc/kubernetes"
 
+	PKISecretName               = "kubernetes-pki"
```

Comment:
So as @luxas mentioned, this is kind of a security hole today b/c secrets are not encrypted in storage.  RBAC fends off some of this, but still.